lib/active_merchant/billing/credit_card.rb

```diff
@@ -103,6 +105,7 @@ def number=(value)
       # * +'alia'+
       # * +'olimpica'+
       # * +'creditel'+
+      # * +'confiable'+
```

Comment:
Just as a side-comment not related to your PR, I think it is weird that we have two separate lists of supported card types in this one file.

Reply:
I agree that it is weird. It is so weird that when I added Creditel I missed one of the lists :P